addons/folo/common/src/main/java/org/commonjava/indy/folo/change/FoloTrackingListener.java

```diff
@@ -64,13 +66,14 @@
 
     public void onFileAccess( @Observes final FileAccessEvent event )
     {
-        final TrackingKey trackingKey = (TrackingKey) event.getEventMetadata()
-                                                           .get( FoloConstants.TRACKING_KEY );
+        EventMetadata metadata = event.getEventMetadata();
+        final TrackingKey trackingKey = (TrackingKey) metadata.get( FoloConstants.TRACKING_KEY );
         if ( trackingKey == null )
         {
             logger.info( "No tracking key for access to: {}", event.getTransfer() );
             return;
         }
+        final AccessChannel accessChannel = (AccessChannel) metadata.get( FoloConstants.ACCESS_CHANNEL );
```

Comment:
I think the constant belongs somewhere more generally associated with EventMetadata...maybe even in the EventMetadata class itself...